modules/core/src/main/java/org/apache/ignite/spi/discovery/tcp/TcpDiscoverySpi.java

```diff
@@ -837,7 +844,8 @@ public TcpDiscoverySpi setStatisticsPrintFrequency(long statsPrintFreq) {
     }
 
     /**
-     * Gets IP finder clean frequency.
+     * Gets frequency which coordinator clean IP finder and keeps it in the correct state, unregistering addresses of
+     * the nodes that has left the topology.
```

Comment:
Gets frequency _with_ which coordinator clean_s_ IP finder and keeps it in the correct state, unregistering addresses of
the nodes that _have_ left the topology.